client/my-sites/stats/stats-module/style.scss

```
@@ -586,3 +586,81 @@ ul.module-header-actions {
 .stats__module-list .stats-module__availability-warning-message {
 	font-size: $font-body-extra-small;
 }
+
+.stats-module--unified {
+	.card {
+		&:first-child {
+			border-radius: 4px 4px 0 0;
+			box-shadow: none;
+			border: 1px solid var(--studio-gray-5);
+			border-bottom: 0;
+			margin-bottom: 0;
+			padding: 24px 24px 12px;
+		}
+
+		&:last-child {
+			border-radius: 0 0 4px 4px;
+			box-shadow: none;
+			border: 1px solid var(--studio-gray-5);
+			border-top: 0;
+		}
+
+		&:first-child:last-child {
+			border-radius: 4px;
+		}
+	}
+
+	.card__link-indicator {
+		top: 6px;
```

```suggestion
		display: none;
```

For chevron removal as requested by Jeff.